### PR TITLE
Add Jacobi tutorial to documentation

### PR DESCRIPTION
**Pull request by GeliezaK**
Added a tutorial on computing the 1D Jacobi method in parallel with PartitionedArrays to the documentation as a getting started page.

### CODE REVIEW

**Review by fverdugo — COMMENTED** [review]
Hi @GeliezaK 

the tutorial looks very good!

The documentation build is failing. Run it locally and try to reproduce the error? Maybe something related to mpi?

**Comment by GeliezaK** [comment]
Hi @fverdugo, 
the issue did not appear when running it locally. So I have worked around it by putting the code that was executed with mpi in a markdown code block.

**Comment by fverdugo** [comment]
Hi @GeliezaK 

Excellent Job!

Some minor comments:

- The output of @show and error messages (like when you index ranks[1]) do not show in the final page. Pleas rephrase the corresponding parts taking into account this. E.g, Perhaps remove the @show.

- Since we plan to add more tutorials (e.g. spmv), we need to create a new section called "Tutorials", and put this tutorial as the first one.

**Comment by GeliezaK** [comment]
Hi @fverdugo , 
I wrapped the errorneous lines in try-catch-blocks, removed the `@show`, and added the tutorial to a new "Tutorials" section.

File: docs/src/jacobi_tutorial.md
Comment (fverdugo) [inline docs/src/jacobi_tutorial.md:1]: Hi @GeliezaK 

just a final (minor comment).

Please remove this .md file from the repo as it is generated automatically.

**Review by fverdugo — CHANGES_REQUESTED** [review]
See my minor comment.

File: docs/src/jacobi_tutorial.md
Comment (GeliezaK) [reply docs/src/jacobi_tutorial.md:1]: Thanks! I deleted it.

**Comment by fverdugo** [comment]
@GeliezaK Thanks for this PR!

**Pull request merged**